### ISSUE TITLE
Bad CPU type in executable:

### ISSUE THREAD

**Issue by kuncevic** [deanishe/alfred-firefox#35]
Any ideas why it does not work?

[18:51:17.661] ERROR: Firefox Assistant[Script Filter] Code 127: .../Library/Caches/com.runningwithcrayons.Alfred/Workflow Scripts/95DE088A-0C35-4969-9C51-0BEE9ED65DF7:1: bad CPU type in executable: ./alfred-firefox

I am on m1, could it be the case? Does the extension supports m1? 

<img width="217" alt="image" src="https://user-images.githubusercontent.com/1283734/165715542-f5d9d981-3b09-4642-9aa3-e1b8d5a20653.png">

**Comment by kuncevic** [comment]
Yeah [seems like](https://apple.stackexchange.com/questions/408375/zsh-bad-cpu-type-in-executable) this is a case.

**Comment by globau** [comment]
With Rosetta going a way in macOS 28 this will be problematic.

I have an ARM build here: https://github.com/globau/alfred-firefox
